code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -264,6 +264,46 @@
 	name = "Robust Harvest"
 	id = "robustharvest"
 
+/datum/reagent/toxin/fertilizer/monoammoniumphosphate
+	name = "Monoammonium Phosphate"
+	id = "monoammoniumphosphate"
+	strength = 0.25
+	description = "Commonly found in fire extinguishers, also works as a fertilizer."
+	reagent_state = SOLID
+	color = "#999999"
+	taste_description = "salty dirt"
+	metabolism = REM * 2
+
+/datum/reagent/toxin/fertilizer/monoammoniumphosphate/touch_turf(var/turf/simulated/T)
+	if(!istype(T))
+		return
+
+	var/hotspot = (locate(/obj/fire) in T)
+	if(hotspot && !istype(T, /turf/space))
+		var/datum/gas_mixture/lowertemp = T.remove_air(T:air:total_moles)
```

Comment:
Bad use of colon access operators. T is known as a turf. Just replace with periods.